src/App.js

```diff
@@ -5,6 +5,11 @@ import "./App.css";
 
 // Logo has already been provided for you. Do the same for the remaining components
 import Logo from "./components/DisplayComponents/Logo";
+import Display from "./components/DisplayComponents/Display";
+import {Numbers} from "./components/ButtonComponents/NumberButtons/Numbers";
+import {Operators} from "./components/ButtonComponents/OperatorButtons/Operators";
+import {Specials} from "./components/ButtonComponents/SpecialButtons/Specials";
+
```

Comment:
It's conventional to import your components without the curly's i.e. Logo and Display.